src/Components/FooterContainer/__snapshots__/FooterContainer.test.js.snap

```diff
@@ -2,48 +2,48 @@
 
 exports[`<FooterContainer /> should render correctly 1`] = `
```

Comment:
Had to regenerate the footer as the id have changed as a result of upping the packages. Think this is okay - looks okay from the front